build.py

```diff
@@ -45,12 +46,32 @@
                            .replace(MAGIC_BODY, reader.read()) \
                            .replace(MAGIC_CSS, prefix) \
                            .replace(MAGIC_JS, prefix)
-        with open(file_name, 'w') as writer:
+
+        if not os.path.exists(BUILD_FOLDER):
+            os.makedirs(BUILD_FOLDER)
+
+        with open(BUILD_FOLDER + "/" + file_name, 'w') as writer:
             writer.write(html)
+        print(strftime("%Y-%m-%d %H:%M:%S", gmtime()) + ' Finished: ' + file_name)
     if has_modification:
-        with open('sitemap.txt', 'w') as writer:
-            writer.write('https://cyberzhg.github.io/toolbox/\n')
+        print("Modification detected")
+        with open(BUILD_FOLDER + "/" + 'sitemap.txt', 'w') as writer:
+            print("Writing sitemap")
+            writer.write('https://mindfa.onrender.com/\n')
             for file_name in os.listdir(PARTS_FOLDER):
                 if file_name[-5:] != '.html':
                     continue
-                writer.write('https://cyberzhg.github.io/toolbox/' + file_name[:-5] + '\n')
+                writer.write('https://mindfa.onrender.com/' + file_name[:-5] + '\n')
```

Comment:
Note that I changed the URL here, you'll have to change it back to yours! This will be best in a .env.